lib/cqm/converter/qdm_patient.rb

```diff
@@ -2,9 +2,318 @@
 module CQM::Converter
   # CQM Converter class for QDM based patients.
   class QDMPatient
+    # Initialize a new HDSRecord converter. NOTE: This should be done once, and then
```

Comment:
Should this say `QDMPatient` converter, rather than HDSRecord?

Reply:
Indeed it should, fixed.